blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/BlueOceanUrlActionImpl.java

```diff
@@ -0,0 +1,55 @@
+package io.jenkins.blueocean;
+
+import hudson.Extension;
+import hudson.model.ModelObject;
+import io.jenkins.blueocean.rest.factory.BlueOceanUrlAction;
+import io.jenkins.blueocean.rest.factory.BlueOceanUrlMapper;
+
+import javax.annotation.Nonnull;
+
+import static io.jenkins.blueocean.BlueOceanUrlMapperImpl.getLandingPagePath;
```

Comment:
This probably should need to be moved here, as we may not have the dependency with pipeline plugins